src/posts/2022-02-10-design-het-web.md

```diff
@@ -0,0 +1,68 @@
+
+Er zijn een aantal mogelijkheden waar je uit kan kiezen als designer: je wil doorleren en de code echt gaan beheersen, je wilt omscholen en het design achter je laten, je vindt allebei leuk en blijft er tussenin hangen of je sluit de code helemaal uit.
+
+## Vergeet het design
+
+Er is een mogelijkheid om het designen helemaal te laten voor wat het is en je vol te storten op de code. Dit is iets wat ik niet zou aanraden; veel bedrijven zijn blij met developers die ook designkennis hebben. Om binnen organisaties de overlap tussen design en development zo soepel mogelijk te laten lopen zijn een aantal, of tenminste één zo’n persoon nodig. Je kunt dus een belangrijke rol gaan spelen.
+
+In het begin heb je nog niet veel aan je design skills, dit komt omdat je nog heel beperkt bent in wat je kunt maken. In een later stadium zul je gaan merken dat je niet alleen vette dingen kunt maken, maar ook verzinnen. Je kunt dan nauw samenwerken met de designers, en ze zullen ook blij zijn met jou.
+
+## Leer het bij: Code, Code, Code!
+
+In het begin is het niet te overzien; verschillende tools, talen & frameworks; Vue, Angular, React etc!
+Ook worden er veel fancy Engelse namen en code talen gebruikt in dagelijks communicatie. Afhankelijk van je doel is de learning curve best stijl, zeker als je alleen een design achtergrond hebt. Er zijn ook veel overeenkomsten, die zie je vooral terug in HTML/CSS. Denk aan eigenschappen van elementen, werkstructuur en het uiteindelijke doel; gave dingen maken!
+
+Ik heb gemerkt dat het iets is wat je over je heen moet laten komen. Veel stof is logischer en minder ingewikkeld dan hoe het er in eerste instantie uitziet. Als designer heb je ook te maken met projectmatig werken, het einddoel waarborgen en een professioneel product afleveren. De tooling is alleen anders.
+Het moeilijkste om te leren is het denken als een developer; een soort analytische, oplossingsgerichte visie die je moet hebben, naast je algemene programmeer vaardigheden natuurlijk. Met genoeg oefening is dit zeker onder de knie te krijgen en moeten de meeste designers met een beetje geduld, hier wel uit gaan komen.
+
+<figure>
+    <img src="/assets/design-het-web/code.jpg" alt="Code">
+    <figcaption>Leer project HTML/CSS</figcaption>
+</figure>
+
+## De dubbele pet: WordPress
+
+In mijn geval was ik meer vaardig in design terwijl ik meer interesse had voor de development. Als tussenoplossing daarvoor maakte ik gebruik van WordPress.
+Een veel gebruikt CMS die theme based veel mogelijkheden biedt. Na het maken van het design kon ik het in een populaire builder zoals [Avada](https://avada.theme-fusion.com/) of [Elementor](https://elementor.com/) het design ongeveer nabouwen. Het design van het web is voor klein MKB veruit het belangrijkst. De Achterliggende techniek is niet zo belangrijk. Door een behoorlijke groep mensen word de designer ook gezien als developer. Als designer heb je dus een dubbele pet als je het hebt over Theme based werken.
```

Comment:
>De Achterliggende techniek

_achterliggende_?

> groep mensen word de designer
_wordt_?

> die theme based veel
over Theme based werken

Choose one or the other 🙂